tg/decorators.py

```diff
@@ -329,7 +329,42 @@ class decode_params(object):
 
     By default the arguments are parsed in **JSON** format (which is
     currently the only supported format).
-
+    
+    This should be used like:
+        Image you are posting a payload of type 'application/json' like:
+        
+        ticket = {
+          ticketlist_id: 2,
+          typology: 'Xmas Discount',
+          quantity: 3,
+          price: 4.75
+        }
+        
+        Your controller's method should be something like
+        
+        .. code-block:: python
+        
+        @expose('json')
+        @decode_params(format='json')
+        def create(self, **kw):
+            print('*' * 60)
+            print('kw', kw)
+            ...do stuff...
+            return dict(ticket=something)
```

Comment:
I think you can remove this first example and leave around only the second.
We don't have to explain python packing and unpacking of arguments :D

Reply:
As you want, it is the same, it was just one example more